sftk/common.py

```diff
@@ -0,0 +1,53 @@
+import os
+from dotenv import load_dotenv
+
+def load_env_wrapper() -> None:
+    """
+    Wrapper for loading environment variables from a .env file.
+
+    Guard clause added to prevent loading environment variables when running in a GitHub Actions environment.
+    """
+    if os.environ.get("GITHUB_ACTIONS") == "true":
+        return
+
+    load_dotenv()
+
+
+load_env_wrapper()
+
+# Email configuration
+EMAIL_USER = os.getenv("EMAIL_USER")
+EMAIL_PASS = os.getenv("EMAIL_PASS")
+IMAP_SERVER = "imap.gmail.com"
+IMAP_PORT = 993
+EMAIL_ARCHIVE_FOLDER_CANDIDATES = [
+    "[Gmail]/All Mail",  # Gmail
+    "Archive",  # Common IMAP
+    "Archived",  # Alternative naming
+    "INBOX.Archive",  # Some email systems
+]
+EMAIL_ARCHIVE_FOLDER = "[Gmail]/All Mail"
+
+# S3 configuration
+S3_BUCKET = os.getenv("S3_BUCKET")
+AWS_ACCESS_KEY_ID = os.getenv("AWS_ACCESS_KEY_ID")
+AWS_SECRET_ACCESS_KEY = os.getenv("AWS_SECRET_ACCESS_KEY")
+
+S3_SHAREPOINT_MOVIE_CSV = os.getenv("S3_SHAREPOINT_MOVIE_CSV")
+S3_SHAREPOINT_SURVEY_CSV = os.getenv("S3_SHAREPOINT_SURVEY_CSV")
+S3_SHAREPOINT_SITE_CSV = os.getenv("S3_SHAREPOINT_SITE_CSV")
+S3_SHAREPOINT_SPECIES_CSV = os.getenv("S3_SHAREPOINT_SPECIES_CSV")
```

Comment:
Can you share the .env file sample - an .env file with all the entries but no values defined?